### PR TITLE
Fix: Wrong stat values on Matomo

### PR DESCRIPTION
**Pull request by nkokla**

### CODE REVIEW

File: src/hooks/useContactFormFCU.ts [hunk 1]
```diff
@@ -119,13 +120,13 @@ const useContactFormFCU = () => {
   );
 
   const handleOnSubmitContact = useCallback(
-    async (data: Record<string, any> = {}) => {
-      if (data.structure !== 'Tertiaire') {
+    async (data?: AddressDataType) => {
+      if (data && data.structure !== 'Tertiaire') {
         data.company = '';
       }
```
Comment (desoindx) [inline src/hooks/useContactFormFCU.ts:126]: This has no sens... If data is not defined, then data.company will fail...

File: src/types/AddressData.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export type AddressDataType = {
+  address?: string;
+  company?: string;
+  geoAddress?: Record<string, any>;
+  eligibility?: boolean;
+  computEligibility?: boolean;
+  heatingType?: AvailableHeating;
+  network?: Record<string, any>;
+  structure?: AvailableStructure;
+  distance?: number;
```
Comment (desoindx) [inline src/types/AddressData.d.ts:12]: There is a lot of new field in this type, are you sure you're not mixing two types ?

**Review by desoindx — CHANGES_REQUESTED** [review]

File: src/types/AddressData.d.ts (hunk 2 above)
Comment (nkokla) [reply src/types/AddressData.d.ts:12]: No, these are fine all the data that emerges from the eligibility/contact form and who is send from components to components inside an object.

File: src/hooks/useContactFormFCU.ts (hunk 1 above)
Comment (nkokla) [reply src/hooks/useContactFormFCU.ts:126]: I not sure to understand. In this case, `data` is defined. 
(`if (data && ...) data.company = ''`)

Comment (desoindx) [reply src/hooks/useContactFormFCU.ts:126]: my bad !

**Pull request merged**